eng/packages/http-client-csharp/generator/TestProjects/Local/Mgmt-TypeSpec/src/Generated/RestOperations/PrivateLinksRestOperations.cs

```diff
@@ -15,31 +15,23 @@
 
 namespace MgmtTypeSpec
 {
-    /// <summary></summary>
-    public partial class PrivateLinks
+    internal partial class PrivateLinks
     {
         private readonly Uri _endpoint;
-        /// <summary> A credential used to authenticate to the service. </summary>
-        private readonly TokenCredential _tokenCredential;
```

Comment:
Those 2 properties are never used?

Reply:
The credential fields are never used in the sub client since we have HttpPipeline already.
And that's why they should be removed from the generated code.